index.jsx

```diff
@@ -157,7 +157,7 @@ module.exports = function Dimensions ({
         }
         return (
           <div className={className} style={containerStyle} ref='container'>
-            {(containerWidth || containerHeight) &&
+            {!(containerWidth || containerHeight) : null ?
```

Comment:
I understand this is probably pending a maintainer. But just to note, it appears the ternary has the `?` and `:` in the incorrect order.

Reply:
It should at least log a warning in the console (in my opinion), mainly because it'll happen with any animation in the parent container and if you don't know that it's quite messy to debug. A proper solution should be made because if you have animations it will not render your component ever.